templates/manx.html

```diff
@@ -1,12 +1,11 @@
 <meta id="websocket-data" data-websocket="{{ websocket }}">
 
-<script src="/manx/js/xterm.js"></script>
-<script src="/manx/js/terminal.js"></script>
+<script type="module" src="/manx/js/terminal.js"></script>
```

Comment:
When the manx tab is closed, there's an error from this script:
![Screen Shot 2021-11-01 at 5 01 50 PM](https://user-images.githubusercontent.com/6034483/139741736-5deda1ef-68c8-41f2-a533-ccd8801d35bb.png)

If you open the manx tab after, the terminal doesn't load so it requires a page refresh to get it working again.

Reply:
I fixed the error, but the manx terminal will require a refresh if it gets closed and reopened due to how to the script file is loaded :/